fast/stages/02-networking-nva/vpc-spoke-dev.tf

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Dev spoke VPC and related resources.
+
+module "dev-spoke-project" {
+  source          = "../../../modules/project"
+  billing_account = var.billing_account_id
+  # TODO: reset counter
+  name   = "dev-net-spoke-3"
+  parent = var.folder_id
+  prefix = var.prefix
+  service_config = {
+    disable_on_destroy         = false
+    disable_dependent_services = false
+  }
+  services = [
+    "compute.googleapis.com",
+    "dns.googleapis.com",
+    "iap.googleapis.com",
+    "networkmanagement.googleapis.com",
+    "servicenetworking.googleapis.com",
+  ]
+  shared_vpc_host_config = {
+    enabled          = true
+    service_projects = []
+  }
+  metric_scopes = [module.landing-project.project_id]
+  iam = {
+    "roles/dns.admin" = [var.project_factory_sa.dev]
+  }
+}
+
+module "dev-spoke-vpc" {
+  source                          = "../../../modules/net-vpc"
+  project_id                      = module.dev-spoke-project.project_id
+  name                            = "dev-spoke-0"
+  mtu                             = 1500
+  data_folder                     = "${var.data_dir}/subnets/dev"
+  delete_default_routes_on_create = true
+  subnets_l7ilb                   = local.l7ilb_subnets.dev
+  # Set explicit routes for googleapis; send everything else to NVAs
+  routes = {
+    private-googleapis = {
```

Comment:
Nit: customer might want private/restricted traffic to go through IPS. 
Either version is fine, but I'd document this choice (if you haven't already).

Reply:
Ludo reminded me this wouldn't work, so self-resolving :)